recipe/meta.yaml

```diff
@@ -23,20 +23,20 @@ requirements:
     - pkg-config  # [not win]
     - libtool  # [not win]
     - xz 5.2.*  # [not win]
-    - freetype 2.7  # [not win]
+    - freetype 2.8.1  # [not win]
     - fontconfig 2.12.*  # [not win]
-    - glib 2.51.*  # [not win]
+    - glib 2.55.*  # [not win]
     - pixman 0.34.*
-    - libpng >=1.6.22,<1.6.31
+    - libpng >=1.6.32,<1.6.35
     - zlib 1.2.11
     - icu 58.*
     - vc 9  # [win and py27]
     - vc 14  # [win and (py35 or py36)]
   run:
-    - freetype 2.7  # [not win]
+    - freetype 2.8.1  # [not win]
     - fontconfig 2.12.*  # [not win]
-    - libpng >=1.6.22,<1.6.31
-    - glib 2.51.*  # [not win]
+    - libpng >=1.6.32,<1.6.35
```

Comment:
Any reason this doesn't match up with the [`libpng >=1.6.22,<1.6.31` global pinning]( https://github.com/conda-forge/conda-forge.github.io/blob/cbbe486d281aec41c69ca96d09acc4813fa96abb/scripts/pin_the_slow_way.py#L68 )?